### ISSUE TITLE
[Feature Request]: Edit plots that are part of multiple smaller figures

### ISSUE THREAD

**Issue by PerPalmgren** [jasp-stats/jasp-issues#1354]
* JASP version: 0.19.3 - 0.97.1

![IM4FVQCL6U10](https://user-images.githubusercontent.com/79397662/125050362-3006ae00-e0a2-11eb-800e-d6fda0885c4c.png)

Being able to customize if scientific notation is shown and being able to edit multi-plots

**Comment by vandenman** [comment]
possible duplicate of https://github.com/jasp-stats/jasp-issues/issues/1352

The scientific notation is there to avoid the y-axis labels from becoming too large. The next version of JASP will allow you to modify the plot labels and remove the scientific notation if you prefer.

**Comment by PerPalmgren** [comment]
I could not see this new function in 0.15(that I am trying now have the opportunity to try as there has been a problem with the Swedish letters  (å, ä, ö) in the regressions.😎
Per

**Comment by vandenman** [comment]
If you click on the name of a plot:
![image](https://user-images.githubusercontent.com/21319932/125249292-5b350b80-e2f5-11eb-824f-717a0b7ae47c.png)

you should get a menu like this, where you can select plot editing:
![image](https://user-images.githubusercontent.com/21319932/125249341-6d16ae80-e2f5-11eb-847c-fb5341167e42.png)

**Comment by PerPalmgren** [comment]
It seems to work in distributions plots but not in scatter plots.

**Comment by vandenman** [comment]
If you're getting this error message:
![image](https://user-images.githubusercontent.com/21319932/125257871-195c9300-e2fe-11eb-97a6-17d269f08117.png)

then it is not supported (and probably won't be supported for the upcoming release).

**Comment by PerPalmgren** [comment]
Yes! It was this error message..
That is sad it will not be in the new version 😢
Per

Per J. Palmgren | DC, FRCC, MMedEd, PhD |
Associate Professor |
Department of Learning, Informatics, Management and Ethics |
Karolinska Institutet | Tomtebodavägen 18A | 171 77 Stockholm |
Tel. + 46 8 524 85 294<tel:+%2046%208%20524%2085%C2%A0294>
***@***.******@***.***> | ki.se<https://eur01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fki.se%2F&data=04%7C01%7Cper.palmgren%40ki.se%7C9f6969aac91f4526542308d875898009%7Cbff7eef1cf4b4f32be3da1dda043c05d%7C0%7C0%7C637388580700328145%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=f6OY6fEm6z0scmnO%2FDhlWRgL%2BujfjA072A2uHaASfNY%3D&reserved=0>
________________________________

Karolinska Institutet – a medical university

**Comment by AlexanderLyNL** [comment]
One way to hack this is by saving the plot as a pptx-file and edit it in powerpoint or google slides, also available in 0.14.1

https://jasp-stats.org/wp-content/uploads/2020/07/Save_Plot_Powerpoint-1.gif

**Comment by PerPalmgren** [comment]
My concern was primarily for when using JASP in teaching in during online statistic labs.

Per J. Palmgren | DC, FRCC, MMedEd, PhD |
Associate Professor |
Department of Learning, Informatics, Management and Ethics |
Karolinska Institutet | Tomtebodavägen 18A | 171 77 Stockholm |
Tel. + 46 8 524 85 294<tel:+%2046%208%20524%2085%C2%A0294>
***@***.******@***.***> | ki.se<http://ki.se/>
________________________________

Karolinska Institutet – a medical university

12 juli 2021 kl. 12:57 skrev AlexanderLyNL ***@***.***>:

﻿

One way to hack this is by saving the plot as a pptx-file and edit it in powerpoint or google slides, also available in 0.14.1

https://jasp-stats.org/wp-content/uploads/2020/07/Save_Plot_Powerpoint-1.gif<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjasp-stats.org%2Fwp-content%2Fuploads%2F2020%2F07%2FSave_Plot_Powerpoint-1.gif&data=04%7C01%7Cper.palmgren%40ki.se%7C6982c412ad4642f4cc7708d94523cbc3%7Cbff7eef1cf4b4f32be3da1dda043c05d%7C0%7C0%7C637616842306790665%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=wmoG%2FK4wHCNW5HWgQHOHduIyI9tfAv026KqX9L7pH0c%3D&reserved=0>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjasp-stats%2Fjasp-issues%2Fissues%2F1354%23issuecomment-878180287&data=04%7C01%7Cper.palmgren%40ki.se%7C6982c412ad4642f4cc7708d94523cbc3%7Cbff7eef1cf4b4f32be3da1dda043c05d%7C0%7C0%7C637616842306795652%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZChu%2FZotwlmEHjUWbffb08%2BLk6bZZWBBCvD1sJTVLRw%3D&reserved=0>, or unsubscribe<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAS5YGHS7YDTELAEWJ2E2SALTXLDAJANCNFSM5ACKLQBQ&data=04%7C01%7Cper.palmgren%40ki.se%7C6982c412ad4642f4cc7708d94523cbc3%7Cbff7eef1cf4b4f32be3da1dda043c05d%7C0%7C0%7C637616842306800641%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4Zc%2FUD9hHjoO%2FqZgCoRRZeKK6fqB9nAXLH%2FVHHIRI7o%3D&reserved=0>.



När du skickar e-post till Karolinska Institutet (KI) innebär detta att KI kommer att behandla dina personuppgifter. Här finns information om hur KI behandlar personuppgifter<https://ki.se/medarbetare/integritetsskyddspolicy>.


Sending email to Karolinska Institutet (KI) will result in KI processing your personal data. You can read more about KI’s processing of personal data here<https://ki.se/en/staff/data-protection-policy>.

**Comment by tomtomme** [comment]
Also related: https://github.com/jasp-stats/jaspGraphs/pull/72

**Comment by juliuspfadt** [comment]
How is this going? We (@EJWagenmakers) needed something similar for a recent paper, and resorted back to R, @boutinb?

**Comment by EJWagenmakers** [comment]
Just an update: I don't mind the scientific notation that much (it can be circumvented by computing kilo-units for instance) but it would be nice to have an on-off switch in the graph editor. And it would be nice if the graph editor allowed users to zoom in on the main graph element (for multi-element figures) or on a specific graph in a multi-panel plot...

**Comment by boutinb** [comment]
I think @vandenman can say more about the feasibility of such a feature.

**Comment by juliuspfadt** [comment]
but if you switch off the notation then the graph becomes almost undisplayable in some cases. Ideally, we would the ability to also edit the lower level plots in a multi-plot...

**Comment by tomtomme** [comment]
Note:
Idea by EJ: "for some multi-component plots it is evident what the "main" component is. If the code identies that part of the plot as the "main", could the editor then not update the specs for the "main" part only? This would not address issues with a multi-panel scatterplot, but it would allow editing of some of our Bayesian prior-posterior plots with the embellishments on top..."

**Comment by tomtomme** [comment]
@PerPalmgren 
Next jasp version will have a dedicated plot builder module that should help with this.

Of course an upgrade of the general plot editor to handle multi-plots stays on the to-do.

**Comment by juliuspfadt** [comment]
I dont think that this PR https://github.com/jasp-stats/jaspGraphs/pull/157 will fully fix this. It is a huge step forward, but ideally each subplot of a plot with multiple figures would be editable.
